2013-01-08 20:42:05 Error: [MissingControllerException] Controller class PERFILsController could not be found.
#0 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-08 20:52:50 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.u_s_u_a_r_i_o_id' in 'on clause'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFIL`...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFIL`...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1254): DboSource->fetchAll('SELECT `PERFIL`...', false)
#4 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1117): DboSource->queryAssociation(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, true, Array, 0, Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#8 [internal function]: USUARIOsController->view('1')
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 20:53:20 Error: [MissingTableException] Table PERFILUSUARIO for model PERFILUSUARIO was not found in datasource default.
#0 /var/www/sit/src/lib/Cake/Model/Model.php(3180): Model->setSource('PERFILUSUARIO')
#1 /var/www/sit/src/lib/Cake/Model/Model.php(1301): Model->getDataSource()
#2 /var/www/sit/src/lib/Cake/Model/Model.php(847): Model->schema()
#3 /var/www/sit/src/lib/Cake/Model/Model.php(872): Model->__isset('PERFILUSUARIO')
#4 /var/www/sit/src/lib/Cake/Model/Model.php(3248): Model->__get('PERFILUSUARIO')
#5 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1598): Model->joinModel('PERFILUSUARIO', Array)
#6 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1073): DboSource->generateAssociationQuery(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, false, NULL)
#7 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#8 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#9 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#10 [internal function]: USUARIOsController->view('1')
#11 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#12 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#13 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#14 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2013-01-08 20:56:19 Error: [MissingTableException] Table PERFILUSUARIO for model PERFILUSUARIO was not found in datasource default.
#0 /var/www/sit/src/lib/Cake/Model/Model.php(3180): Model->setSource('PERFILUSUARIO')
#1 /var/www/sit/src/lib/Cake/Model/Model.php(1301): Model->getDataSource()
#2 /var/www/sit/src/lib/Cake/Model/Model.php(847): Model->schema()
#3 /var/www/sit/src/lib/Cake/Model/Model.php(872): Model->__isset('PERFILUSUARIO')
#4 /var/www/sit/src/lib/Cake/Model/Model.php(3248): Model->__get('PERFILUSUARIO')
#5 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1598): Model->joinModel('PERFILUSUARIO', Array)
#6 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1073): DboSource->generateAssociationQuery(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, false, NULL)
#7 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#8 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#9 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#10 [internal function]: USUARIOsController->view('1')
#11 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#12 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#13 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#14 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2013-01-08 20:56:20 Error: [MissingTableException] Table PERFILUSUARIO for model PERFILUSUARIO was not found in datasource default.
#0 /var/www/sit/src/lib/Cake/Model/Model.php(3180): Model->setSource('PERFILUSUARIO')
#1 /var/www/sit/src/lib/Cake/Model/Model.php(1301): Model->getDataSource()
#2 /var/www/sit/src/lib/Cake/Model/Model.php(847): Model->schema()
#3 /var/www/sit/src/lib/Cake/Model/Model.php(872): Model->__isset('PERFILUSUARIO')
#4 /var/www/sit/src/lib/Cake/Model/Model.php(3248): Model->__get('PERFILUSUARIO')
#5 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1598): Model->joinModel('PERFILUSUARIO', Array)
#6 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1073): DboSource->generateAssociationQuery(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, false, NULL)
#7 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#8 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#9 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#10 [internal function]: USUARIOsController->view('1')
#11 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#12 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#13 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#14 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2013-01-08 20:56:20 Error: [MissingTableException] Table PERFILUSUARIO for model PERFILUSUARIO was not found in datasource default.
#0 /var/www/sit/src/lib/Cake/Model/Model.php(3180): Model->setSource('PERFILUSUARIO')
#1 /var/www/sit/src/lib/Cake/Model/Model.php(1301): Model->getDataSource()
#2 /var/www/sit/src/lib/Cake/Model/Model.php(847): Model->schema()
#3 /var/www/sit/src/lib/Cake/Model/Model.php(872): Model->__isset('PERFILUSUARIO')
#4 /var/www/sit/src/lib/Cake/Model/Model.php(3248): Model->__get('PERFILUSUARIO')
#5 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1598): Model->joinModel('PERFILUSUARIO', Array)
#6 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1073): DboSource->generateAssociationQuery(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, false, NULL)
#7 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#8 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#9 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#10 [internal function]: USUARIOsController->view('1')
#11 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#12 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#13 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#14 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2013-01-08 20:56:21 Error: [MissingTableException] Table PERFILUSUARIO for model PERFILUSUARIO was not found in datasource default.
#0 /var/www/sit/src/lib/Cake/Model/Model.php(3180): Model->setSource('PERFILUSUARIO')
#1 /var/www/sit/src/lib/Cake/Model/Model.php(1301): Model->getDataSource()
#2 /var/www/sit/src/lib/Cake/Model/Model.php(847): Model->schema()
#3 /var/www/sit/src/lib/Cake/Model/Model.php(872): Model->__isset('PERFILUSUARIO')
#4 /var/www/sit/src/lib/Cake/Model/Model.php(3248): Model->__get('PERFILUSUARIO')
#5 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1598): Model->joinModel('PERFILUSUARIO', Array)
#6 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1073): DboSource->generateAssociationQuery(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, false, NULL)
#7 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#8 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#9 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#10 [internal function]: USUARIOsController->view('1')
#11 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#12 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#13 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#14 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#15 {main}
2013-01-08 20:56:41 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.u_s_u_a_r_i_o_id' in 'on clause'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFIL`...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFIL`...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1254): DboSource->fetchAll('SELECT `PERFIL`...', false)
#4 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1117): DboSource->queryAssociation(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, true, Array, 0, Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#8 [internal function]: USUARIOsController->view('1')
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 20:56:57 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.u_s_u_a_r_i_o_id' in 'on clause'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFIL`...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFIL`...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1254): DboSource->fetchAll('SELECT `PERFIL`...', false)
#4 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1117): DboSource->queryAssociation(Object(USUARIO), Object(PERFILUSUARIO), 'hasAndBelongsTo...', 'PERFIL', Array, Array, true, Array, 0, Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#8 [internal function]: USUARIOsController->view('1')
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 20:57:19 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.u_s_u_a_r_i_o_id' in 'on clause'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFIL`...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFIL`...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1254): DboSource->fetchAll('SELECT `PERFIL`...', false)
#4 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1117): DboSource->queryAssociation(Object(USUARIO), Object(PERFILUSUARIO), 'hasAndBelongsTo...', 'PERFIL', Array, Array, true, Array, 0, Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#8 [internal function]: USUARIOsController->view('1')
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 20:57:59 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.u_s_u_a_r_i_o_id' in 'on clause'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFIL`...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFIL`...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1254): DboSource->fetchAll('SELECT `PERFIL`...', false)
#4 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1117): DboSource->queryAssociation(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, true, Array, 0, Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#8 [internal function]: USUARIOsController->view('1')
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 20:58:21 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.u_s_u_a_r_i_o_id' in 'on clause'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFIL`...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFIL`...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1254): DboSource->fetchAll('SELECT `PERFIL`...', false)
#4 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1117): DboSource->queryAssociation(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, true, Array, 0, Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#8 [internal function]: USUARIOsController->view('1')
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 20:58:22 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.u_s_u_a_r_i_o_id' in 'on clause'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFIL`...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFIL`...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1254): DboSource->fetchAll('SELECT `PERFIL`...', false)
#4 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1117): DboSource->queryAssociation(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, true, Array, 0, Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#8 [internal function]: USUARIOsController->view('1')
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 20:58:22 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.u_s_u_a_r_i_o_id' in 'on clause'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFIL`...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFIL`...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1254): DboSource->fetchAll('SELECT `PERFIL`...', false)
#4 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1117): DboSource->queryAssociation(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, true, Array, 0, Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#8 [internal function]: USUARIOsController->view('1')
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 20:58:22 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.u_s_u_a_r_i_o_id' in 'on clause'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFIL`...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFIL`...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1254): DboSource->fetchAll('SELECT `PERFIL`...', false)
#4 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1117): DboSource->queryAssociation(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, true, Array, 0, Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '1')
#8 [internal function]: USUARIOsController->view('1')
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 21:00:22 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.p_e_r_f_i_l_id' in 'field list'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFILU...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFILU...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1082): DboSource->fetchAll('SELECT `PERFILU...', false)
#4 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(AppModel), Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(1847): Model->find('all', Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1743): Model->_saveMulti(Array, '7', Object(Mysql))
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(43): Model->save(Array)
#8 [internal function]: USUARIOsController->add()
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 21:01:18 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'prueba@sit.com' for key 'correo_UNIQUE'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('INSERT INTO `go...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1009): DboSource->execute('INSERT INTO `go...')
#3 /var/www/sit/src/lib/Cake/Model/Model.php(1730): DboSource->create(Object(USUARIO), Array, Array)
#4 /var/www/sit/src/app/Controller/USUARIOsController.php(43): Model->save(Array)
#5 [internal function]: USUARIOsController->add()
#6 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#7 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#8 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#9 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2013-01-08 21:01:27 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.p_e_r_f_i_l_id' in 'field list'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFILU...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFILU...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1082): DboSource->fetchAll('SELECT `PERFILU...', false)
#4 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(AppModel), Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(1847): Model->find('all', Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1743): Model->_saveMulti(Array, '9', Object(Mysql))
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(43): Model->save(Array)
#8 [internal function]: USUARIOsController->add()
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 21:01:56 Error: [PDOException] SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry 'prueba2@sit.com' for key 'correo_UNIQUE'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('INSERT INTO `go...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1009): DboSource->execute('INSERT INTO `go...')
#3 /var/www/sit/src/lib/Cake/Model/Model.php(1730): DboSource->create(Object(USUARIO), Array, Array)
#4 /var/www/sit/src/app/Controller/USUARIOsController.php(43): Model->save(Array)
#5 [internal function]: USUARIOsController->add()
#6 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#7 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#8 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#9 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#10 {main}
2013-01-08 21:02:03 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.p_e_r_f_i_l_id' in 'field list'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFILU...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFILU...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1082): DboSource->fetchAll('SELECT `PERFILU...', false)
#4 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(AppModel), Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(1847): Model->find('all', Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1743): Model->_saveMulti(Array, '11', Object(Mysql))
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(43): Model->save(Array)
#8 [internal function]: USUARIOsController->add()
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 21:02:18 Error: [PDOException] SQLSTATE[42S22]: Column not found: 1054 Unknown column 'PERFILUSUARIO.u_s_u_a_r_i_o_id' in 'on clause'
#0 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(461): PDOStatement->execute(Array)
#1 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(427): DboSource->_execute('SELECT `PERFIL`...', Array)
#2 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(671): DboSource->execute('SELECT `PERFIL`...', Array, Array)
#3 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1254): DboSource->fetchAll('SELECT `PERFIL`...', false)
#4 /var/www/sit/src/lib/Cake/Model/Datasource/DboSource.php(1117): DboSource->queryAssociation(Object(USUARIO), Object(PERFIL), 'hasAndBelongsTo...', 'PERFIL', Array, Array, true, Array, 0, Array)
#5 /var/www/sit/src/lib/Cake/Model/Model.php(2631): DboSource->read(Object(USUARIO), Array)
#6 /var/www/sit/src/lib/Cake/Model/Model.php(1516): Model->find('first', Array)
#7 /var/www/sit/src/app/Controller/USUARIOsController.php(32): Model->read(NULL, '4')
#8 [internal function]: USUARIOsController->view('4')
#9 /var/www/sit/src/lib/Cake/Controller/Controller.php(485): ReflectionMethod->invokeArgs(Object(USUARIOsController), Array)
#10 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(186): Controller->invokeAction(Object(CakeRequest))
#11 /var/www/sit/src/lib/Cake/Routing/Dispatcher.php(161): Dispatcher->_invoke(Object(USUARIOsController), Object(CakeRequest), Object(CakeResponse))
#12 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#13 {main}
2013-01-08 21:09:40 Error: Fatal Error (64): Cannot use [] for reading in [/var/www/sit/src/app/Controller/UsuarioController.php, line 47]
2013-01-08 21:09:40 Error: [FatalErrorException] Cannot use [] for reading
#0 /var/www/sit/src/lib/Cake/Error/ErrorHandler.php(162): ErrorHandler::handleFatalError(64, 'Cannot use [] f...', '/var/www/sit/sr...', 47)
#1 [internal function]: ErrorHandler::handleError(64, 'Cannot use [] f...', '/var/www/sit/sr...', 47, Array)
#2 /var/www/sit/src/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 64, 'Cannot use [] f...', '/var/www/sit/sr...', 47, Array)
#3 /var/www/sit/src/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-08 21:10:53 Error: Fatal Error (1): Call to a member function login() on a non-object in [/var/www/sit/src/app/Controller/UsuarioController.php, line 17]
2013-01-08 21:10:53 Error: [FatalErrorException] Call to a member function login() on a non-object
#0 /var/www/sit/src/lib/Cake/Error/ErrorHandler.php(162): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/sit/sr...', 17)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/sit/sr...', 17, Array)
#2 /var/www/sit/src/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/sit/sr...', 17, Array)
#3 /var/www/sit/src/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-08 21:12:54 Error: Fatal Error (1): Call to a member function login() on a non-object in [/var/www/sit/src/app/Controller/UsuarioController.php, line 17]
2013-01-08 21:12:54 Error: [FatalErrorException] Call to a member function login() on a non-object
#0 /var/www/sit/src/lib/Cake/Error/ErrorHandler.php(162): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/sit/sr...', 17)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/sit/sr...', 17, Array)
#2 /var/www/sit/src/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/sit/sr...', 17, Array)
#3 /var/www/sit/src/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-08 21:14:50 Error: Fatal Error (1): Call to a member function login() on a non-object in [/var/www/sit/src/app/Controller/UsuarioController.php, line 17]
2013-01-08 21:14:50 Error: [FatalErrorException] Call to a member function login() on a non-object
#0 /var/www/sit/src/lib/Cake/Error/ErrorHandler.php(162): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/sit/sr...', 17)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/sit/sr...', 17, Array)
#2 /var/www/sit/src/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/sit/sr...', 17, Array)
#3 /var/www/sit/src/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-08 21:16:32 Error: Fatal Error (1): Call to a member function login() on a non-object in [/var/www/sit/src/app/Controller/UsuarioController.php, line 19]
2013-01-08 21:16:32 Error: [FatalErrorException] Call to a member function login() on a non-object
#0 /var/www/sit/src/lib/Cake/Error/ErrorHandler.php(162): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/sit/sr...', 19)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/sit/sr...', 19, Array)
#2 /var/www/sit/src/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/sit/sr...', 19, Array)
#3 /var/www/sit/src/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-08 21:27:12 Error: Fatal Error (1): Call to a member function login() on a non-object in [/var/www/sit/src/app/Controller/UsuarioController.php, line 19]
2013-01-08 21:27:12 Error: [FatalErrorException] Call to a member function login() on a non-object
#0 /var/www/sit/src/lib/Cake/Error/ErrorHandler.php(162): ErrorHandler::handleFatalError(1, 'Call to a membe...', '/var/www/sit/sr...', 19)
#1 [internal function]: ErrorHandler::handleError(1, 'Call to a membe...', '/var/www/sit/sr...', 19, Array)
#2 /var/www/sit/src/lib/Cake/Core/App.php(926): call_user_func('ErrorHandler::h...', 1, 'Call to a membe...', '/var/www/sit/sr...', 19, Array)
#3 /var/www/sit/src/lib/Cake/Core/App.php(899): App::_checkFatalError()
#4 [internal function]: App::shutdown()
#5 {main}
2013-01-08 21:30:06 Error: [MissingControllerException] Controller class LoginController could not be found.
#0 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-08 21:30:15 Error: [MissingControllerException] Controller class LoginController could not be found.
#0 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-08 21:30:31 Error: [MissingControllerException] Controller class LoginController could not be found.
#0 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
2013-01-08 21:30:32 Error: [MissingControllerException] Controller class LoginController could not be found.
#0 /var/www/sit/src/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
